sqlstore/liquidity_provision.go

```diff
@@ -126,8 +126,7 @@ func (lp *LiquidityProvision) buildLiquidityProvisionsSelect(partyID entities.Pa
 	reference string) (string, []interface{}) {
 	var bindVars []interface{}
 
-	selectSql := fmt.Sprintf(`select distinct on (id) %s
-from liquidity_provisions`, sqlOracleLiquidityProvisionColumns)
+	selectSql := fmt.Sprintf(`select %s from liquidity_provisions_current`, sqlOracleLiquidityProvisionColumns)
```

Comment:
Just a thought here, in the migration file, the SQL is written all in caps, here's it all lower case. Would be good to have 1 way of writting SQL over the whole codebase.